index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header id="header" >
+        <span id="logo">Abdullah Asghar</span>
+        <button type="button" class="btn">
+            <img src="./icon/Icon-Menu.png" alt="Menu-icon">
+        </button>
+    </header>
+    
+    <main>
+        <section id="Headline">
+            <h1>Hey There.<br>I’m Abdullah</h1>
+            <h4 >I’am a Software Developer</h4>
+            <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.
+            </p>
+
+                <div class="Social-media-icons">
+                    <a href="#">
+                        <img src="./icon/Disabled.png" alt="-GitHub icon">
+                    </a>
+                    <a href="#">
+                        <img src="./icon/Disabled1.png" alt="Linkedin icon">
+                    </a>
+                    <a href="#">
+                        <img src="./icon/Enabled.png" alt="twenty icon">
+                    </a>
+                    <a href="#">
+                        <img src="./icon/Disabled2.png" alt="Twitter icon">
+                    </a>
+                    <a href="#">
+                        <img src="./icon/Disabled3.png" alt="M icon">
+                    </a>
+                </div>
```

Comment:
kindly the social media should me in the listed, like :
` <ul> <li>...</li></ul>
`